src/features/click-to-load.js

```diff
@@ -367,33 +354,11 @@ class DuckWidget {
     }
 }
 
-function replaceTrackingElement (widget, trackingElement, placeholderElement, hideTrackingElement = false, currentPlaceholder = null) {
+function replaceTrackingElement (widget, trackingElement, placeholderElement, currentPlaceholder = null) {
```

Comment:
I'm not sure if this is caused by any of these changes, it probably isn't, but Log In with Facebook is not working for me.
- https://privacy-test-pages.glitch.me/privacy-protections/click-to-load/ : The modal appears for the first button, but then nothing happens. For the second button, it gets stuck with "Calling FB SDK..."
![image](https://user-images.githubusercontent.com/14364481/228269444-0f5bff05-decb-4264-912f-c55901e0a748.png)

- https://en.boardgamearena.com/account?page=newuser&src=login : The custom FB button doesn't work
- https://www.babyartikel.de/en/login: Our FB button gets replaced only after clicking the second time

Reply:
The first one is expected.

I'm not sure about the other two. I can reproduce, but I can also reproduce with the released extension (and before these changes.) I will open a task to investigate.

Reply:
OK, yea second one is an unrelated issue that I've now pinned down. We need to start redirecting requests `https://connect.facebook.net/en_US/all.js` instead of blocking them. I'm not sure about the last website, but the Facebook login flow there seems kind of broken anyway.

Reply:
Thank you for taking a look into this! 🙌  I'll resolve this since it's unrelated to this PR.